knock/src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from './Profile.module.scss';
+import profileImage from '../../core/assets/image/Profilephoto.svg';
+import ShareButtons from '../../core/ui/buttons/ShareButtons/ShareButtons';
+
+interface ProfileProps {}
+
+const Profile = ({}: ProfileProps) => {
+  return (
+    <div className={`${styles['container']}`}>
+      <img
+        src={profileImage}
+        alt="profileImage"
+        className={`${styles['profileImage']}`}
+      />
+      <div className={`${styles['text']}`}>아초는고양이</div>
```

Comment:
이름 부분 Props로 내려받는 걸 빼먹으셨어요! 😄
```suggestion
      <div className={`${styles['text']}`}>{프롭스로_받은_이름}</div>
```